test.js

```diff
@@ -979,11 +979,4 @@ describe('is-binary', function() {
   it('should be a function', function() {
     isBinary.should.be.a('function');
   });
-  it('should correctly determine binary files', function() {
```

Comment:
These passed before I removed them. No point in having them here as they're in `is-binary-path`.